com.claudiocambra.brickbuster.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  - SPDX-License-Identifier: GPL-3.0-or-later
+  - SPDX-FileCopyrightText: 2021 Claudio Cambra <claudio.cambra@gmail.com>
+ -->
+<component type="desktop">
+  <id>com.claudiocambra.brickbuster</id>
+  <provides>
+    <binary>brickbuster_linux.x86_64</binary>
+  </provides>
+  <name>BrickBuster</name>
+  <summary>A brick-breaker game, fun and addictive!</summary>
+  <description>
+    <p>BrickBuster is a brick-breaker type game where the objective is to survive as many rounds as possible. Every round new bricks appear... when they reach the bottom of the screen, you lose. You are armed with balls and any special nodes that luck sends your way.</p>
+  </description>
+  <url type="homepage">https://github.com/claucambra/BrickBuster</url>
+  <url type="bugtracker">https://github.com/claucambra/BrickBuster/issues</url>
+  <categories>
+    <category>Games</category>
+  </categories>
+  <developer_name>Claudio Cambra</developer_name>
+  <metadata_license>GPL-3.0</metadata_license>
```

Comment:
GPL-3.0 isn't an accepted license for the appstream metadata. CC0-1.0 is usually preferred.

Reply:
Noted, changed to CC0-1.0